src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1632,7 +1633,25 @@ function ($a) {
                         ->arrayNode('stop_worker_on_signals')
                             ->defaultValue([])
                             ->info('A list of signals that should stop the worker; defaults to SIGTERM and SIGINT.')
-                            ->integerPrototype()->end()
+                            ->beforeNormalization()
+                                ->always()
+                                ->then(function ($v) {
+                                    return array_map(static function ($element): int|string {
+                                        $element = \is_array($element) ? $element['signal'] : $element;
```

Comment:
why this support for arrays with a `signal` key ?

Reply:
This is an artifact, it's removed :slightly_smiling_face: